src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -31,6 +32,7 @@ public static Person[] getSamplePersons() {
                     new Email("alexyeoh@example.com"),
                     new Address("Blk 30 Geylang Street 29, #06-40"),
                     new Gender("Male"),
+                    new GraduationDate("05-2022"),
```

Comment:
For the sample data, I think it's better to put future dates and a variety of dates e.g. `12-2022`, `05-2023`, or `12-2023` because these will be the first data the users view